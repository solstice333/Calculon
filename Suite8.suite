P a.out main.c header.h
T test12.in  test12.out 1000 -a -b -c
P b.out foo.c
T test13.in  test13.out 1000 -a
P c.out main2.c
T test14.in  test14.out 1000 -cat
