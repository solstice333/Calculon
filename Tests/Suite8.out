a.out 1 of 1 tests passed.
foo.c: In function ‘main’:
foo.c:7: error: ‘This’ undeclared (first use in this function)
foo.c:7: error: (Each undeclared identifier is reported only once
foo.c:7: error: for each function it appears in.)
foo.c:7: error: expected ‘;’ before ‘Is’
Failed: gcc foo.c -o b.out
c.out 1 of 1 tests passed.
